src/shared/logToSentry.ts

```diff
@@ -0,0 +1,26 @@
+import { z } from 'zod';
+// other details like timestamp, and hostname/ IP address should be included by Sentry
+const zLogObject = z.object({
+    message: z.string(),
+    email: z.string().email().optional(),
+    data: z.object({}).optional()
+})
+
+enum logLevel{
+    ERROR,
+    INFO
+}
+
+export type logObject = z.TypeOf<typeof zLogObject>;
+
+const logToSentry = (tag: keyof typeof logLevel, logDetail: logObject) => {
```

Comment:
Why do you need the `LogObject` type? Why don't  just pass message, email, data as three function parameters?

Reply:
I was thinking of creating this object type just in case if we need more keys or properties to be passed to log without changing the number of parameters of `logToSentry`, but I did a quick research about the inefficiency of JSON.stringfy and will change it to a list parameters